src/astro/utils/dataframe_function_handler.py

```diff
@@ -0,0 +1,87 @@
+import inspect
+from abc import ABC
+from typing import Callable, Dict, Optional, Tuple
+
+import pandas as pd
+
+from astro.databases import create_database
+from astro.databases.base import BaseDatabase
+from astro.sql.table import Table
+
+
+class DataframeFunctionHandler(ABC):
+    """Contains functions for converting to dataframe or converting from dataframe"""
+
+    database_impl: BaseDatabase
+    output_table: Optional[Table]
+    op_args: Tuple
+    op_kwargs: Dict
+    python_callable: Callable
+    identifiers_as_lower: bool = False
+    conn_id: str = ""
+
+    def load_op_arg_dataframes_into_sql(self):
+        """Identifies dataframes in op_args and loads them to the table"""
+        final_args = []
+        for arg in self.op_args:
+            if isinstance(arg, pd.DataFrame):
+                self.database_impl.load_pandas_dataframe_to_table(
+                    source_dataframe=arg, target_table=self.output_table
```

Comment:
Shouldn't we create a new table object here? otherwise, this will keep updating the same output table.